index.html

```diff
@@ -4,6 +4,13 @@
   <meta charset="utf-8">
   <meta name="viewport" content="target-densitydpi=device-dpi, height=660, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
   <title>Blockly Demo: Blockly Developer Tools</title>
+  <script>
+  // For importing outside libraries.
+  if (typeof module === 'object') {
+    window.module = module;
+    module = undefined;
```

Comment:
This looks suspect to me.  I'd like to dive deeper, and maybe find a better solution.